stylesheets/desktop/stylesheet-blog.css

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  font-family: "Georgia", serif;
+}
+
+header {
+  background-color: #093C60;
+  padding: 5% 0% 5% 0%;
+  position: relative;
+}
+
+.header-intro {
+  margin-left: 10%;
+}
+
+.header-checkout {
+  position: absolute;
+  right: 0%;
+  top: 7%;
+}
+
+.header-logo {
+  width: 30%;
+}
+
+.header-title {
+  color: #81B741;
+  font-size: 1.5em;
+  width: 40%;
+}
+
+.cart-logo {
+  width: 10%;
+}
+
+#header-price {
+  color: white;
+  display: inline-block;
+}
+
+.button {
+  position:absolute;
+  right: 8%;
+  top: 20%;
+  background-color: #81B741;
+  border: none;
+  color: white;
+  padding: 1% 3%;
+  text-align: center;
+  text-decoration: none;
+  font-size: 1em;
+}
+
+.global-navigation {
+  display: flex;
+  font-size: 1em;
+  font-weight: bold;
+  background: #ECECEC;
+  padding: 2% 0% 2% 0%;
+}
+
+.nav-link {
+  color: #093C60;
+  text-decoration: none;
+  padding: 0% 1% 0 15%;
+}
+
+.nav-link-2 {
+  color: #093C60;
+  text-decoration: none;
+  padding: 0% 1% 0 1%;
+}
+
+#blog-header {
+  color: #093C60;
+  font-size: 2em;
+  margin: 2% 0% 2% 15%;
+}
+
+#blog-1 {
+  display: flex;
+  flex-direction: row;
+}
+
+#blog-2 {
```

Comment:
At smaller resolutions the margins are gone since you applied them to the image but not the paragraph. it might be a better idea to add a margin bottom to the blog-#'s so they have some spacing between them.